packages/dataviews/src/dataviews-layouts/grid/preview-size-picker.tsx

```diff
@@ -12,6 +12,10 @@ import DataViewsContext from '../../components/dataviews-context';
 import type { ViewGrid } from '../../types';
 
 const imageSizes = [
+	{
+		value: 120,
+		breakpoint: 1,
+	},
```

Comment:
I actually think we might need another size between 120 and 230, the gap is too big.